WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1254,7 +1256,8 @@ public boolean onPrepareOptionsMenu(Menu menu) {
             // if content has blocks or empty, offer the switch to Gutenberg. The block editor doesn't have good
             //  "Classic Block" support yet so, don't offer a switch to it if content doesn't have blocks. If the post
             //  is empty but the user hasn't enabled "Use Gutenberg for new posts" App setting, don't offer the switch.
-            switchToGutenbergMenuItem.setVisible(hasBlocks || (AppPrefs.isGutenbergDefaultForNewPosts() && isEmpty));
+            switchToGutenbergMenuItem.setVisible(
```

Comment:
Let's also update the comment in the above line (line number 1258) to mention "Site setting" instead of "App setting".

Reply:
Fixed with 1a06e9111892b2bc155af716215b342b0bb0a57f.